packages/atst/README.md

```diff
@@ -0,0 +1,25 @@
+# @eth-optimism/atst (WIP)
+
+WIP sdk and cli tool for the attestation station. Currently in design phase. Any code here is purely experimental proof of concepts.
+
+## Dev setup
+
+To be able to run the cli commands with npx you must first link the node module so npm treats it like an installed package
+
+```
+npm link
+```
+
+Alternatively `yarn dev` will run the cli
```

Comment:
Should this go before the link? as is this the main way to dev? and alternatively people can manually link it to use it with npx